runtime/error.c

```diff
@@ -1,9 +1,116 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
 #include "balrt.h"
+#include "third-party/libbacktrace/backtrace.h"
+
+#define MIN_FRAME_COUNT 2
```

Comment:
It is the initial capacity not the minimum count, isn't it.

Reply:
Yes, Changed